core/src/main/java/org/apache/sdap/mudrod/weblog/structure/ApacheAccessLog.java

```diff
@@ -15,55 +15,49 @@
 
 import com.google.gson.Gson;
 
+import org.apache.sdap.mudrod.main.MudrodConstants;
+import org.apache.sdap.mudrod.weblog.pre.CrawlerDetection;
+
 import java.io.IOException;
 import java.io.Serializable;
 import java.text.ParseException;
 import java.text.SimpleDateFormat;
 import java.util.Date;
+import java.util.Properties;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
-import org.apache.sdap.mudrod.weblog.pre.CrawlerDetection;
-
 /**
  * This class represents an Apache access log line. See
  * http://httpd.apache.org/docs/2.2/logs.html for more details.
  */
 public class ApacheAccessLog extends WebLog implements Serializable {
-
-
-  /**
-   * 
-   */
-  private static final long serialVersionUID = 1L;
-
-  public ApacheAccessLog() {
-    //default constructor
-  }
-
-  String response;
-  String referer;
-  String browser;
+  String Response;
```

Comment:
Unless these are private then they should be below the default constructor!
Revert the global variable and constructor positioning

Reply:
Additionally, Java variable declatations should be ```firstLowerCamelCase``` NOT ```FirstCapitalCamelCase```

Reply:
I suggest creating a new issue about lower/upper case. If I just change them to lower case here, lots of other code will be affected

Reply:
No it will not, you are essentially REVERTING the correct Java code convention here. The changes are restructured to this file.

Reply:
These variables were all upper case at the beginning. The previous enhancement PR changed it to lower case which can result in a null-pointer error, as they correspond to the field names in Elasticsearch which he didn't change. Therefore, I changed it back for the time being as these filed names are used explicitly at many places.